src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -227,7 +227,8 @@ public override object VisitSqlRoot(SqlRootExpression expression, SearchOptions
                     else if (IsSortValueNeeded(searchOptions))
                     {
                         StringBuilder
-                            .Append($"{TableExpressionName(_tableExpressionCounter)}.SortValue ")
+                            .Append(TableExpressionName(_tableExpressionCounter))
+                            .Append(".SortValue ")
```

Comment:
This changes are caused by how interpolated strings work in .net6. Current code can't be compiled and it talks about out parameter somewhere inside string internals.

I could figure out how to make it work, or I could just rewrite to not use interpolated string, and I choose later.